src/server/Uploader.js

```diff
@@ -221,7 +221,8 @@ class Uploader {
 
   uploadTus () {
     const fname = path.basename(this.options.path)
-    const metadata = Object.assign({ filename: fname }, this.options.metadata || {})
+    const ftype = this.options.metadata.type
+    const metadata = Object.assign({ filename: fname, filetype: ftype }, this.options.metadata || {})
```

Comment:
I'd say we should do `delete metadata.type` after this line. But I'd merge and do that.

Reply:
Why delete, garbage collection?

Reply:
no because we'd also be unnecessarily sending the metadata `type` to tusd, and I'm not sure that's intended.